osmoutils/coin_helper.go

```diff
@@ -114,3 +114,11 @@ func MergeCoinMaps[T comparable](currentEpochExpectedDistributionsOne map[T]sdk.
 	}
 	return newMap
 }
+
+func ConvertCoinArrayToCoins(coinArray []sdk.Coin) sdk.Coins {
```

Comment:
While coin arrays can be used in place of a coins object and still be valid, coin array do not posses the same sub methods (ex. Sub, Add, etc), so this method is useful imo.